'Nan::Callback::Call': was declared deprecated

```
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  addon.cc
  process.cc
..\src\addon.cc(88): warning C4996: 'Nan::Callback::Call': was declared deprecated [C:\projects\vscode-windows-process-tree\build\windows_process_tree.vcxproj]
  C:\projects\vscode-windows-process-tree\node_modules\nan\nan.h(1618): note: see declaration of 'Nan::Callback::Call'
  worker.cc
  process_commandline.cc
..\src\worker.cc(54): warning C4996: 'Nan::Callback::Call': was declared deprecated [C:\projects\vscode-windows-process-tree\build\windows_process_tree.vcxproj]
  C:\projects\vscode-windows-process-tree\node_modules\nan\nan.h(1618): note: see declaration of 'Nan::Callback::Call'
```

--- a/src/addon.cc
+++ b/src/addon.cc
@@ -85,7 +85,8 @@ void GetProcessCpuUsage(const Nan::FunctionCallbackInfo<v8::Value>& args) {
   }
 
   v8::Local<v8::Value> argv[] = { result };
-  callback->Call(1, argv);
+  Nan::AsyncResource resource("windows-process-tree:addon.GetProcessCpuUsage");
+  callback->Call(1, argv, &resource);
 }
 
 void Init(v8::Local<v8::Object> exports) {

--- a/src/worker.cc
+++ b/src/worker.cc
@@ -51,5 +51,6 @@ void Worker::HandleOKCallback() {
   }
 
   v8::Local<v8::Value> argv[] = { result };
-  callback->Call(1, argv);
+  Nan::AsyncResource resource("windows-process-tree:worker.HandleOKCallback");
+  callback->Call(1, argv, &resource);
 }

